### DISCUSSION TITLE
using devils sight in sight limited darkness template

### DISCUSSION THREAD

**Discussion by Terrox** [dev7355608/perfect-vision#331] [General]
I use a darkness template to block vision with flags:{'perfect-vision':{sightLimit: 0}}
can I do something to let the devil's sight player see in this darkness?

**Comment by dev7355608** [comment]
Not possible at the moment (see #326/https://github.com/dev7355608/perfect-vision/issues/197#issuecomment-1368504657). But I'm working on a module that will make it possible.

**Comment by dev7355608** [comment]
I initially assumed that you use Adequate Vision, but that might not be the case. With Adequate Vision you can give the player the Devil's Sight detection mode, which lets them see tokens inside and behind the darkness (but not anything else).

**Reply by Terrox** [reply]
I don't, I can't seem to install that one in v9

**Reply by dev7355608** [reply]
Adequate Vision isn't V9 compatible.
